src/clj/s3_beam/handler.clj

```diff
@@ -48,14 +49,14 @@
    "sa-east-1"      "s3-sa-east-1"})
 
 (defn sign-upload [{:keys [file-name mime-type]}
-                   {:keys [bucket aws-zone aws-access-key aws-secret-key]}]
+                   {:keys [bucket aws-zone aws-access-key aws-secret-key acl] :or {acl "public-read"}}]
```

Comment:
There are two places where a `public-read` default is provided. Probably one of both is enough?